collector/teamcity/examples/compose/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.7'
+services:
+  teamcity-server:
+    container_name: teamcity-server
+    image: jetbrains/teamcity-server:latest
+    ports:
+      - "8111:8111"
+    volumes:
+      - ./data/teamcity_server/datadir:/data/teamcity_server/datadir
+      - ./logs:/opt/teamcity/logs
+
+  teamcity-agent:
+    container_name: teamcity-agent
+    image: jetbrains/teamcity-agent:latest
+    environment:
+      - SERVER_URL=http://teamcity-server:8111
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
```suggestion
```
No need to mount docker socket for scraping a prometheus endpoint

Reply:
Hmmm ok. Make sense.